src/lib.rs

```diff
@@ -303,7 +303,7 @@ pub enum RenderNodeInfo {
     /// An unordered list
     Ul(Vec<RenderNode>),
     /// An ordered list
-    Ol(Vec<RenderNode>),
+    Ol(Option<u64>, Vec<RenderNode>),
```

Comment:
I think this would be better as `i64` instead of `Option<u64>`:
* Negative numbers are allowed (but it's possible they wouldn't format properly - I'm happy to fix that later)
* I can't see any reason to distinguish between `<ol>` and `<ol start="1">`, and that slightly simplifies the renderer.